parkhangs-expressapp/scripts/populateFacilities.js

```diff
@@ -0,0 +1,60 @@
+const axios = require("axios")
+const Facility = require('../models/FacilityModel')
+const Park = require('../models/ParkModel')
+
+ // used to populate the park's facilities on the database.
+ // Should not be called unless parks collection on database is removed
+
+const populateFacilities = async function() {
+    try {
+        const parks = await Park.updateMany({}, {"$unset": {"facilities": ""}})
+    } catch (error) {
+        console.log('could not reset all park facilities')
+    }
+    try {
+        let res = await axios.get("https://opendata.vancouver.ca/api/records/1.0/search/?dataset=parks-facilities&q=&rows=700&facet=facilitytype")
```

Comment:
not sure if there would be a performance increase by getting less rows? There should be far less than 700.

Reply:
oh I just wasn't sure how many facilities there would be! not super worried about performance since it's only run once

Reply:
@gordonzhang17 lol just checked the response and it's 633 rows so not super far off!